Frontend/ui-driver/src/Flow.purs

```diff
@@ -679,6 +679,95 @@ driverProfileFlow = do
       modifyScreenState $ BookingOptionsScreenType (\bookingOptions -> bookingOptions{data{vehicleType = state.data.driverVehicleType, vehicleNumber = state.data.vehicleRegNumber, vehicleName = state.data.vehicleModelName, vehicleCapacity = state.data.capacity, downgradeOptions = ((downgradeOptionsConfig state.data.vehicleSelected) <$> state.data.downgradeOptions)}})
       bookingOptionsFlow
 
+    DRIVER_ALTERNATE_CALL_API1 updatedState -> do
```

Comment:
`DRIVER_ALTERNATE_CALL_API1` replace this with something like  `DRIVER_ALTERNATE_CALL_API_ONE` or something meaningful

Reply:
DRIVER_ALTERNATE_CALL_API1 and DRIVER_ALTERNATE_CALL_API_ONE is same @Vignesh-772 where `_API1` looks better than _ONE